btrack/io/_localization.py

```diff
@@ -1,11 +1,13 @@
 from __future__ import annotations
 
-import inspect
+import dataclasses
 import logging
-from typing import Generator, List, Optional, Tuple, Union
+from multiprocessing.pool import ThreadPool as Pool
```

Comment:
Have you considered `concurrent.futures.ThreadPoolExecutor`?

Reply:
Oops - this is a good catch, it's actually supposed to be just a `Pool`

Reply:
I feel like this is something we can take a look at when this is integrated into the napari plugin - I'm not sure what the best way to play with that is yet